insolar/bus/bus.go

```diff
@@ -211,6 +210,7 @@ func (b *Bus) sendTarget(
 
 	// configure logger
 	ctx, _ = inslogger.WithField(ctx, "sending_type", msgType)
+	ctx, _ = inslogger.WithField(ctx, "sending_uuid", msg.UUID)
 	logger := inslogger.FromContext(ctx)
```

Comment:
Can you combine 213 and 214? 
`ctx, logger := inslogger.WithField(ctx, "sending_uuid", msg.UUID)`